packages/ses/rollup.config.js

```diff
@@ -1,30 +1,24 @@
 import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import { terser } from 'rollup-plugin-terser';
-import fs from 'fs';
-
-const metaPath = new URL('package.json', import.meta.url).pathname;
-const meta = JSON.parse(fs.readFileSync(metaPath, 'utf-8'));
-const name = meta.name.split('/').pop();
-const umd = meta.umd || name;
```

Comment:
I’ve removed these because the addition of the `ses/lockdown` build targets has made the `rollup.config.js` for `ses` sufficiently different from other packages that there is no longer an incentive to make them all the same. It is no longer sensible to have a generalized rollup.config.js that we keep in sync across packages.